src/components/v5/common/ActionSidebar/partials/forms/ManageVerifiedMembersForm/utils.ts

```diff
@@ -43,3 +51,36 @@ export const getValidationSchema = (
 export type ManageVerifiedMembersFormValues = InferType<
   ReturnType<typeof getValidationSchema>
 >;
+
+export const getManageVerifiedMembersPayload = (
+  colony: Colony,
+  values: ManageVerifiedMembersFormValues,
+) => {
+  const members = values.members?.map((member) => member?.value);
+
+  const baseDomainPayload = {
+    operation:
+      values.manageMembers === ManageMembersType.Add
+        ? ManageVerifiedMembersOperation.Add
+        : ManageVerifiedMembersOperation.Remove,
+    colonyAddress: colony.colonyAddress,
+    colonyName: colony.name,
+    members,
+    customActionTitle: values.title,
+    annotationMessage: values.description,
+    domainId: values.createdIn || Id.RootDomain,
```

Comment:
a leftover from before, but we don't need `domainId` as this motion/action will always take place in `Id.RootDomain`